apps/andi/lib/andi/input_schemas/struct_tools.ex

```diff
@@ -71,6 +71,12 @@ defmodule Andi.InputSchemas.StructTools do
     struct(struct_type, preloaded)
   end
 
+  defp sort_if_sequenced([%{sequence: _sequence} | _] = list) do
+    Enum.sort(list, fn %{sequence: sequence1}, %{sequence: sequence2} -> sequence1 < sequence2 end)
```

Comment:
Is there a reason why `Enum.sort_by(&Map.get(&1, :sequence))` doesn't work here?

Reply:
Probably would. I'll look into it